src/keyValue.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2021, USC
+// All rights reserved.
+#ifndef SRC_KEYVALUE_H_
+#define SRC_KEYVALUE_H_
+#include <unordered_map>
+#include <vector>
+
+namespace backend {
+
+// TODO: CHANGE BYTES TYPE
+typedef char BYTES;
```

Comment:
What is this for?

Reply:
I was confused about how bytes worked in grpc, but @adamegyed explained to me today that it compiles to a stl string in C++, so I will change char to std::string.